src/main/resources/stsjorbsmodResources/localization/spa/JorbsMod-Power-Strings.json

```diff
@@ -299,8 +299,8 @@
   "stsjorbsmod:ShrapnelBloomPower": {
     "NAME": "Flor de Metralla",
     "DESCRIPTIONS": [
-      "Este turno, la próxima carta no-Legendario se juega %1d vez adicional, después lo #yExpira.",
-      "Este turno, la próxima carta no-Legendario se juega %1d veces adicionales, después lo #yExpira."
+      "Este turno, la próxima carta no- [REMOVE_SPACE]#yLegendario se juega #b%1d vez adicional, después la #yExpira.",
+      "Este turno, la próxima carta no- [REMOVE_SPACE]#yLegendario se juega #b%1d veces adicionales, después la #yExpira."
```

Comment:
by adding `no-legendaria` as a valid keyword for `Legendary`, this can also probably change.